pages/letter/index.html

```diff
@@ -145,13 +145,16 @@ <h3>Thanks!</h3>
     <div class="mzp-l-content mzp-t-content-xl mzp-t-content-nospace">
       <header>
         <aside class="c-signature-count">
-          <strong>1567</strong> signatures
+          <strong>{{ signatures.length }}</strong> signatures
         </aside>
 
         <h2 class="c-signatures-title mzp-u-title-sm">Signatories<span>(in order of signing)</span></h2>
       </header>
 
       <ol class="c-signatures-list">
+      {% for sig in signatures %}
+        <li><cite>{{ sig.name }}</cite>, {{ sig.affiliation }}</li>
```

Comment:
Not all imported signatures have affiliations so we'll want an `if` check on that comma.